compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/split/types/JvmObjectTypeGen.java

```diff
@@ -183,6 +180,16 @@ public void populateObject(ClassWriter cw, MethodVisitor mv, String methodName,
         }
     }
 
+    private List<BAttachedFunction> getObjectAttachedFunctions(BObjectTypeSymbol objectTypeSymbol) {
+        List<BAttachedFunction> attachedFunctions = new ArrayList<>();
+        BAttachedFunction initFunction = objectTypeSymbol.initializerFunc;
+        if (initFunction != null && initFunction.funcName.value.contains("init")) {
```

Comment:
Why we need to check contains "init"

Reply:
This check is already there, need to investigate the necessity of it.